go/arrow/internal/flatbuf/RecordBatch.go

```diff
@@ -128,8 +128,42 @@ func (rcv *RecordBatch) Compression(obj *BodyCompression) *BodyCompression {
 }
 
 /// Optional compression of the message body
+/// Some types such as Utf8View are represented using a variable number of buffers.
```

Comment:
I wonder why `///` is used instead of `//`

Reply:
You'd have to ask the maintainers of `flatc` lol